packages/aws-cdk/lib/api/cxapp/stacks.ts

```diff
@@ -403,3 +412,39 @@ export interface Tag {
   readonly Key: string;
   readonly Value: string;
 }
+
+function _makeCdkMetadataAvailableCondition() {
+  const options = new Array<any>();
+  // Find out all regions known to support AWS::CDK::Metadata from the RegionInfo facts database
+  for (const region of regionInfo.Fact.regions) {
+    if (regionInfo.Fact.find(region, regionInfo.FactName.CDK_METADATA_RESOURCE_AVAILABLE) === 'YES') {
```

Comment:
Would be nice to make this check a bit more strong typed... Something like `regionInfo.cdkMetadataAvailable` (seems like this condition is also duplicated above)

Reply:
You missed this comment...

Reply:
Ah - I only forgot to... push it out. 🤣